miscoptions.go

```diff
@@ -253,3 +253,35 @@ func (n *NII) UnmarshalBinary(b []byte) error {
 
 	return nil
 }
+
+type RelayMessageOption []byte
```

Comment:
Please add a documentation comment.  Same for all other methods added to this file.